### DISCUSSION TITLE
Feature selection in online learning fashion

### DISCUSSION THREAD

**Discussion by Haoyu-R** [online-ml/river#509] [Ideas]
Hi there, 

I just came up with an idea: we can implement some online feature selection methods to eliminate/exclude some irrelevant features before calling the real online-ML algorithm. Also, feature selection can be used as an element in the pipeline of the river. This will improve the overall efficiency of online-ML. 

Some references:
https://arxiv.org/abs/1511.09263
https://www.sciencedirect.com/science/article/abs/pii/S0950705117304124

Cheers.

**Comment by MaxHalford** [comment]
Hey! We've already started a `feature_selection` module, see [here](https://riverml.xyz/latest/api/overview/#feature_selection). But feel free to improve it and/or contribute more methods!
